packages/ripple-binary-codec/src/enums/definitions.json

```diff
@@ -1484,6 +1535,16 @@
         "type": "Vector256"
       }
     ],
+    [
+      "TokenIDs",
+      {
+        "nth": 4,
+        "isVLEncoded": true,
```

Comment:
What does `isVLEncoded` mean?

Reply:
https://xrpl.org/serialization.html#length-prefixing

it's a way of including the length of the array in the binary serialization